loki-build-image/Dockerfile

```diff
@@ -7,6 +7,14 @@ RUN apk add --no-cache curl && \
     mv /tmp/linux-amd64/helm /usr/bin/helm && \
     rm -rf /tmp/linux-amd64 /tmp/helm-$HELM_VER.tgz
 
+FROM alpine as lychee
```

Comment:
Q: why `lychee?`

Reply:
That's the only one I found that is not nodejs and ruby. Which both requires to install the VM to run those and that's just a nono.